packages/components/src/FilterBar/subcomponents/ClearAllButton/ClearAllButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { Button } from "~components/Button"
+import { useFilterBarContext } from "../../context/FilterBarContext"
+import styles from "./ClearAllButton.module.scss"
+
+export const ClearAllButton = (): JSX.Element => {
+  const { clearAllFilters } = useFilterBarContext()
+
+  return (
+    <Button
```

Comment:
@dougmacknz would this need an `aria-controls` since it manipulates the filters?